internal/schema/schema.go

```diff
@@ -44,8 +44,9 @@ func (o SchemaQualifiedName) IsEmpty() bool {
 }
 
 type Schema struct {
-	Tables  []Table
-	Indexes []Index
+	Extensions []Extension
```

Comment:
[You should normalize the extensions like we do here](https://github.com/stripe/pg-schema-diff/blob/3464752b7a32ac83b825bec61a0390cd972bf468/internal/schema/schema.go#L75) in case the queries happen to fetch the extensions in a different order, leading to different hashes for the same schema